src/isIsogram.test.js

```diff
@@ -6,4 +6,44 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('playgrounds', () => {
```

Comment:
it should be like summary in bug report. It should contain short description what u checking in u test